web_external/ImagesGallery/Listing/imagesPagingPane.jade

```diff
@@ -1,45 +1,63 @@
 .growing.flexColumn
-  #isic-images-pagingTools.flexRow
-    #isic-images-seekFirst.button
-      img(src=staticImageRoot + '/seekFirst.svg')
-    #isic-images-seekPrev.button
-      img(src=staticImageRoot + '/seekPrev.svg')
-    #isic-images-seekNext.button
-      img(src=staticImageRoot + '/seekNext.svg')
-    #isic-images-seekLast.button
-      img(src=staticImageRoot + '/seekLast.svg')
-    #isic-images-loading.detailLabel Loading images...
-    #isic-images-noPagingOrFilters.detailLabel
-      | Showing all&nbsp;
-      span.page 0
-      |  images in the archive.
-    #isic-images-hasPaging.detailLabel
-      | Showing images&nbsp;
-      span.page 0 - 0
-      |  of&nbsp;
-      span.filteredSet 0
-      |  total images.
-    #isic-images-hasFilters.detailLabel
-      | Showing&nbsp;
-      span.filteredSet 0
-      |  of the&nbsp;
-      span.overview 0
-      |  total number of images in the archive.
-    #isic-images-hasFiltersAndPaging.detailLabel
-      | Showing images&nbsp;
-      span.page 0 - 0
-      |  of&nbsp;
-      span.filteredSet 0
-      |  filtered images. There are&nbsp;
-      span.overview 0
-      |  total images in the archive (ignoring filters).
-    button#isic-images-download-zip.btn.btn-sm.btn-default(
+  #isic-images-paging-tools.flexRow
+    #isic-images-paging-seek.btn-group
+      button#isic-images-paging-seek-first.btn.btn-sm.btn-default(
+        data-toggle='tooltip'
+        data-placement='bottom'
+        title='First page'
+      )
+        i.icon-to-start-alt
+      button#isic-images-paging-seek-prev.btn.btn-sm.btn-default(
+        data-toggle='tooltip'
+        data-placement='bottom'
+        title='Previous page'
+      )
+        i.icon-fast-bw
+      button#isic-images-paging-seek-next.btn.btn-sm.btn-default(
+        data-toggle='tooltip'
+        data-placement='bottom'
+        title='Next page'
+      )
+        i.icon-fast-fw
+      button#isic-images-paging-seek-last.btn.btn-sm.btn-default(
+        data-toggle='tooltip'
+        data-placement='bottom'
+        title='Last page'
+      )
+        i.icon-to-end-alt
```

Comment:
My buttons start wrapping to a new line if the text beside it is long enough and my window isn't wide enough. I don't think that used to happen.